### DISCUSSION TITLE
Distinguishing Merge Commits?

### DISCUSSION THREAD

**Discussion by oll-gregtasonis** [community/community#56153] [New to GitHub]
How would you go about identifying a "merge commit"? I want to check the commit comments on both parents.

**Comment by airtower-luna** [comment]
"Has more than one parent" is exactly the identifying property for merge commits. :slightly_smiling_face: 

How to check that depends on how you're evaluating the commits. In `git log` output there's conveniently a `Merge:` line listing the parent commits.

```
commit 61de0380f114419c8a0507ea9f5c4c6fafe2ae3b
Merge: 034cb78 131a2af
[...]
```
